Cargo.toml

```diff
@@ -28,19 +28,19 @@ bench = false
 test = true
 
 [dependencies]
-num-integer = "0.1.39"
-num-traits = { version = "0.2", default-features = false }
+num-integer = { version = "0.1.39", default-features = false }
+num-traits = { version = "0.2", default-features = false , features = ["libm"] }
```

Comment:
This adds a dependency to the default configuration, will have to look at that

Reply:
We do need some methods in Float( ln() exp() etc), I think the best way is to use libm features. Or is there any way to detect [default-features] first and then introduce crates?

Reply:
The exclusion of the float trait (already merged) assumed we do it without libm, so I think we can continue on that path. What would break if we don't have libm? We need num-traits with std, when ndarray/std is enabled.

Reply:
Ok, i will try this way. Thanks for reviwing!

Reply:
Is it usable for ndarray's purposes? Users can add features to num that they need for their purposes. :)

Reply:
I'm afraid it is. Will be used in src/geomspace.rs:36, in Iterator for Geomspace.

Reply:
FloatCore is public. It's in num-trait, it's not our trait.

Then we just exclude geomspace when std is not enabled, for now

Reply:
sry it's my mistake. It works :) I will do this way. Thanks a lot!